### PR TITLE
#34 ref: Refactor pa11y script

### PR DESCRIPTION
**Pull request by Abibubble**
### Issue fixed

- #34 

### Description of changes

- Tidy a11y script

### Checks

1. [x] Have you followed the guidelines in our Contributing document?
2. [x] Have you checked to ensure there aren't other open Pull Requests for the same update/change?
3. [x] Have you linted your code locally prior to submission?
4. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
5. [x] Have you checked for any spelling or grammatical mistakes?

### CODE REVIEW

File: scripts/pa11y.js [hunk 1]
```diff
@@ -33,22 +31,24 @@ async function runPa11y(url) {
...
-    // Output the raw result object
-    for (let i = 0; i < result.issues.length; i++) {
-      console.log(colors.red('A11Y ERROR! ' + result.issues[i].message))
+    result.issues.forEach(issue => {
+      console.log(colors.red(`A11Y ERROR! ${issue.message}`))
+      console.log(colors.brightCyan(`The offending code: ${issue.context}`))
       console.log(
-        colors.brightCyan('The offending code: ' + result.issues[i].context)
+        colors.brightMagenta(
+          '           ================================================================          '
```
Comment (DaveyJH) [inline scripts/pa11y.js]: NOPE!!!

**Review by DaveyJH — CHANGES_REQUESTED** [review]
Nice changes, make it better!

File: scripts/pa11y.js [hunk 2]
```diff
@@ -33,22 +31,24 @@ async function runPa11y(url) {
     } else {
       console.log(
         colors.yellow(
-          'There are ' + result.issues.length + ' issues with this page:'
+          `There ${result.issues.length == 1 ? 'is' : 'are'} ${
```
[the remark this answers is not in the document]
Comment (Abibubble) [reply scripts/pa11y.js]: Fixed it! 😄

File: scripts/pa11y.js (hunk 1 above)
Comment (DaveyJH) [reply scripts/pa11y.js]: 🧐

File: scripts/pa11y.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (DaveyJH) [reply scripts/pa11y.js]: Yay!

**Review by DaveyJH — APPROVED** [review]
Fantastic! 🌞

**Pull request merged**